### DISCUSSION TITLE
Interpreting Eval Results

### DISCUSSION THREAD

**Discussion by aflah02** [ContinualAI/avalanche#1185] [Q&A] [answered]
Hi

Can someone help me understand what each of the values represent:

```
-- >> End of eval phase << --
	Loss_Stream/eval_phase/test_stream/Task000 = 0.7185
	Loss_Stream/eval_phase/test_stream/Task001 = 0.7171
	Loss_Stream/eval_phase/test_stream/Task002 = 0.7180
	Top1_Acc_Stream/eval_phase/test_stream/Task000 = 0.5000
	Top1_Acc_Stream/eval_phase/test_stream/Task001 = 0.5000
	Top1_Acc_Stream/eval_phase/test_stream/Task002 = 0.5000
```

My interpretation so far is the first 3 values are the loss of the model on those datasets and the next 3 are the accuracies but I'm not fully sure. Also what does `Top1_Acc_Stream` mean?

**Comment by AntonioCarta** [comment, marked as answer]
ACC=accuracy
Top1=refers to the [top-k accuracy](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.top_k_accuracy_score.html)
Stream=tells you that the accuracy is computed for the entire stream that you have given for evaluation. Alternative are averages of the single experience or minibatch.

**Reply by aflah02** [reply]
Thanks!
